site/content/blogs/release-v130.en.md

```diff
@@ -38,6 +38,16 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
 
 ## Ctrl-C
 
+When your Copilot Service is failing to deploy, You can now hit `Ctrl-C` to rollback your current CloudFormation stack to previous configuration.
+If you hit 'Ctrl-C' again during a rollback, the rollback of CloudFormation stack will continue in the background.
+
+If you are deploying Copilot service for the first time in an environment, hitting `Ctrl-C` during deployment will delete the stack.
+If you hit 'Ctrl-C' again during a stack deletion, the deletion of CloudFormation stack will continue in the background.
+
+You can also now use `--detach` flag to skip the progress of Cloudformation Stack Events on your terminal.   
```

Comment:
I sort of think this way deserve a separate section 💭  It's sort of a separarte feature than `ctrl+c`